lua/clangd_extensions/switch_source_header.lua

```diff
@@ -14,10 +14,12 @@ local function handler(_err, uri)
     }, {})
 end
 
-return {
```

Comment:
we don't need this now, there is no point in restructuring because it _might_ be extended

Reply:
> we don't need this now, there is no point in restructuring because it _might_ be extended

Done, reverted commit.